docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -117,24 +115,39 @@ az get-vm-ip MyResourceGroup MyVM
 
 You can also use environment variables in commands invoked by aliases, which are evaluated at runtime. The next example adds the `create-rg` alias, which creates a resource group in `eastus` and adds an `owner` tag. This tag is assigned the value of the local environment variable `USER`.
 
+```azurecli
+az alias create \
+    --name "create-rg {{ groupName }}" \
+    --command "group create --name {{ groupName }} --location eastus --tags owner=\$USER"
 ```
-[create-rg {{ groupName }}]
-command = group create --name {{ groupName }} --location eastus --tags owner=$USER
-```
+
+To register the environment variables inside the command of the alias, the dollar sign `$` needs to be escaped by adding a slash `\` in front of it.
```

Comment:
Leave out the instructions on how to escape the `$` token, and just leave it as `the variable must be escaped`. This is important since shell escape is only needed inside `""` strings and users may write them as `''`.